.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Is there a valid reason for this? Because I could see that AndroidStudio most of the times ping-pongs between 7 and 8, which one is "better"?

Reply:
I don't know why Android Studio did it, however `languageLevel="JDK_1_8"` seems more coherent with `project-jdk-name="1.8"`, right after it